scss/_patterns_navigation.scss

```diff
@@ -292,19 +291,21 @@ $lightness-threshold: 70;
   background-color: $color-navigation-background;
 
   %navigation-link-theme {
+    position: relative;
```

Comment:
If this is needed (doesn't seem to) it shouldn't be in the theme, but in the component styling.

Reply:
not needed for this version of the design